code/modules/mob/living/living.dm

```diff
@@ -634,6 +634,7 @@ below 100 is not dizzy
 		return
 
 	update_sight()
+	animate(client, time = 0, pixel_x = 0, pixel_y = 0)
```

Comment:
Actually shouldnt this be an animate() with no args

Reply:
it doesn't work without client, pixel+x, and pixel_y but i've removed time = 0. thanks